src/ansys/dpf/composites/data_sources.py

```diff
@@ -67,6 +98,12 @@ class CompositeDataSources:
     """Provides data sources related to the composite lay-up."""
 
     rst: DataSources
```

Comment:
Shouldn't that become a list as well?

Reply:
No, a single `DataSources` object refers to all distributed RST files (added via either `set_result_file_path` for a single file, or `set_domain_result_file_path` for multiple RST).